HIPs/solang.md

```diff
@@ -47,13 +47,18 @@ Currently, solang targets the following blockchains:
 At the time of writing, these chains are the 2nd, 9th, and 11th by market
 capitalization according to [coinmarketcap](https://coinmarketcap.com/).
 
+% How did you find these rankings?
```

Comment:
I couldn't find them. It would be nice to add the direct link to them or a quick explanation about how you got them.